src/NServiceBus.Core.Tests/Sagas/SagaMetadataCreationTests.cs

```diff
@@ -56,6 +56,16 @@ public void When_finder_for_non_message()
         }
 
         [Test]
+        public void When_message_only_has_custom_finder()
+        {
+            var availableTypes = new List<Type>
+            {
+                typeof(SagaWithFinderOnly.Finder)
+            };
+            SagaMetadata.Create(typeof(SagaWithFinderOnly), availableTypes, new Conventions());
```

Comment:
some assertion is missing here. Were you thinking about using `Assert.ShouldNotThrow`?